aten/src/ATen/native/Normalization.cpp

```diff
@@ -103,11 +103,7 @@ namespace {
 template<typename T>
 struct InvStd {
   T operator()(T var, double epsilon) const {
-    T invstd = 0;
-    if (var != static_cast<T>(0) || epsilon != static_cast<T>(0)) {
-      invstd = static_cast<T>(1) / std::sqrt(var + epsilon);
-    }
-    return invstd;
+    return static_cast<T>(1) / std::sqrt(var + epsilon);
```

Comment:
hmm, why do you change things like this?

Reply:
CUDA directly calculates `c10::cuda::compat::rsqrt(var + eps)`.
I do this change to align it with cuda.
If var + eps <= 0 is a meaningless input and we check `var + eps <= 0`,  users might not be able to aware that the input is meaningless.

Reply:
Change it back.